stage1/init/pod.go

```diff
@@ -506,7 +513,20 @@ func (p *Pod) appToNspawnArgs(ra *schema.RuntimeApp) ([]string, error) {
 			opt[0] = "--bind="
 		}
 
-		opt[1] = vol.Source
+		if vol.Kind == "host" {
```

Comment:
Hmm, not sure why we aren't more stringent here already about Kind, but we should be
```
switch vol.Kind {
case "host": ...
case "empty": ...
default: error
}
```